Products/DataCollector/zenmodeler.py

```diff
@@ -158,10 +160,23 @@ def connected(self):
         """
         Called after connected to the zenhub service
         """
+        reactor.callLater(_CONFIG_PULLING_TIMEOUT, self._checkConfigLoad)
         d = self.configure()
         d.addCallback(self.heartbeat)
         d.addErrback(self.reportError)
 
+    
+    def _checkConfigLoad(self):
+        """
+        Looping call to check whether zenmodeler got configuration
+        from ZenHub.
+        """
+        if not self.configLoaded:
+            self.log.info("Modeling is not started yet as zenmodeler"
+            " trying to pull configuration from ZenHub. Please check ZenHub, is it overloaded?")
```

Comment:
Also, this code will result in
```
Modeling is not started yet as zenmodeler trying to pull configuration ZenHub. Please check ZenHub, is it overloaded?
```
as the output.  Perhaps this phrasing reads better:
```
    self.log.info(
        "Modeling has not started pending configuration "
        "pull from ZenHub. Is ZenHub overloaded?"
    )
```